spark/src/test/scala/geotrellis/spark/filter/RasterRDDCropMethodsSpec.scala

```diff
@@ -0,0 +1,39 @@
+package geotrellis.spark.filter
+
+import geotrellis.spark._
+import geotrellis.spark.filter._
+import geotrellis.raster.io.geotiff.SingleBandGeoTiff
+import geotrellis.vector.Extent
+
+import org.scalatest.FunSpec
+
+
+class RasterRDDCropMethodsSpec extends FunSpec with TestEnvironment {
+
+  describe("RasterRDD Crop Methods") {
+    val path = "raster-test/data/aspect.tif"
+    val gt = SingleBandGeoTiff(path)
+    val originalRaster = gt.raster.resample(500, 500)
+    val (_, rdd) = createRasterRDD(originalRaster, 5, 5, gt.crs)
+    val md = rdd.metadata
+    val overall = md.extent
+    val half = Extent(633750, 218375, 641250, 225125)
+    val small = Extent(639000.0, 217700.0, 642000.0, 220399.0)
```

Comment:
Should do computation to get new extents, in the unlikely event that the extent of the underlying raster would ever change.